gitbook-docs/market/economics.md

```diff
@@ -1,124 +1,181 @@
 # Economics
 
-{% hint style="danger" %}
-### Disclaimer
+{% hint style="info" %}
```

Comment:
I really don't think it should be a callout block. A page shouldn't start with a callout, because a callout is what interrupts the story with some important information. But this is just a normal intro. Hello, this document is about this and that.